scapy/main.py

```diff
@@ -176,7 +176,8 @@ def list_contrib(name=None):
         if mod.endswith(".py"):
             mod = mod[:-3]
         desc = { "description":"-", "status":"?", "name":mod }
-        for l in open(f):
+        from io import open
```

Comment:
Can you move this import statement to the beginning of the file?

Reply:
Done !